packaging/installer/methods/kubernetes.md

```diff
@@ -19,94 +20,119 @@ The Helm chart installs one `parent` pod for storing metrics and managing alarm
 pods/containers, and [supported application-specific
 metrics](https://github.com/netdata/helmchart#service-discovery-and-supported-services).
 
+### Prerequisites
+
 To deploy Kubernetes monitoring with Netdata, you need:
 
--   A working cluster running Kubernetes v1.9 or newer.
--   The [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) command line tool, within [one minor version
+- A working cluster running Kubernetes v1.9 or newer.
+- The [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) command line tool, within [one minor version
     difference](https://kubernetes.io/docs/tasks/tools/install-kubectl/#before-you-begin) of your cluster, on an
     administrative system.
--   The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+- The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+- A Netdata Cloud account with a Space to claim the cluster to.
 
-## Install the Netdata Helm chart
+## Deploy Netdata on your Kubernetes Cluster
 
-We recommend you install the Helm chart using our Helm repository. In the `helm install` command, replace `netdata` with
-the release name of your choice.
+To start [Kubernetes monitoring](https://github.com/netdata/netdata/blob/master/docs/cloud/visualize/kubernetes.md), you must first install Netdata, and then
```

Comment:
Too subtle to have the link here. Make it more explicit with a sentence in the introductory line. (something like to see what you'll get, read xxx).

Note also that the linked file has an invalid link back to this one AND repeats installation prerequisites instructions. Please add another PR to fix that.